### PR TITLE
Update authors

### PR DESCRIPTION
**Pull request by joestringer**
Update authors/mailmap for the latest group of contributors.

### CODE REVIEW

**Review by sayboras — APPROVED** [review]
It's awesome to see more and more contributors from various companies 💯

**Review by tklauser — APPROVED** [review]

File: AUTHORS [hunk 1]
```diff
@@ -316,10 +357,12 @@ Pierre-Yves Aillet                      pyaillet@gmail.com
 Pranavi Roy                             pranvyr@gmail.com
 Pratyush Singhal                        psinghal20@gmail.com
 Qasim Sarfraz                           qasim.sarfraz@esailors.de
+qifeng guo                              qifeng.guo@daocloud.io
```
Comment (aanm) [inline AUTHORS]: Qifeng Guo?

File: AUTHORS [hunk 2]
```diff
@@ -412,19 +461,29 @@ Wang Li                                 wangli09@kuaishou.com
...
 Weizhou Lan                             weizhou.lan@daocloud.io
+wenhuwang                               wang15691700816@gmail.com
 Wenxian Li                              wofanli@gmail.com
+Will Daly                               widaly@microsoft.com
 Will Deuschle                           wdeuschle@palantir.com
 Will Stewart                            will@northflank.com
+Wojtek Czekalski                        me@wczekalski.com
 Wongyu Lee                              kyu21@outlook.com
+Xiang Liu                               lx1036@126.com
+xiaoqing                                xiaoqingnb@gmail.com
```
Comment (aanm) [inline AUTHORS]: Xiao Qing?

File: AUTHORS [hunk 3]
```diff
@@ -412,19 +461,29 @@ Wang Li                                 wangli09@kuaishou.com
...
+xiaoyang zhu                            zhuxiaoyang1996@gmail.com
```
Comment (aanm) [inline AUTHORS]: Xiao Yang Zhu?

File: AUTHORS [hunk 4]
```diff
@@ -412,19 +461,29 @@ Wang Li                                 wangli09@kuaishou.com
 Wayne Haber                             whaber@gitlab.com
 Wazir Ahmed                             wazir@accuknox.com
 Weilong Cui                             cuiwl@google.com
+Wei Yang                                wei.yang@daocloud.io
 Weizhou Lan                             weizhou.lan@daocloud.io
+wenhuwang                               wang15691700816@gmail.com
```
Comment (aanm) [inline AUTHORS]: ‪Wenhui Wang?

File: AUTHORS (hunk 2 above)
Comment (joestringer) [reply AUTHORS]: I don't know if this is a "first" name or two names together or the order the names go in. I can capitalize based on naive english grammar rules, but I'm not sure whether I should be picking some specific split/order/capitalization of this name in general.

File: AUTHORS (hunk 3 above)
Comment (joestringer) [reply AUTHORS]: I don't know if this is correct, but I don't mind falling back to naive english grammar rules here (Xiaoyang Zhu).

File: AUTHORS (hunk 4 above)
Comment (joestringer) [reply AUTHORS]: not sure where the `i` comes from, but `Wenhu Wang` seems like a reasonable approximation.

**Pull request merged**